osc-server/src/main/java/org/osc/core/broker/service/securityinterface/BaseSecurityGroupInterfaceService.java

```diff
@@ -53,18 +54,21 @@ protected VirtualSystem validateAndLoad(EntityManager em, SecurityGroupInterface
             throw new VmidcBrokerValidationException("Security group interfaces cannot be created or updated for appliance manager that does not support policy mapping.");
         }
 
-        Policy policy = PolicyEntityMgr.findById(em, dto.getPolicyId());
+        for(PolicyDto policyDto : dto.getPolicies()){
```

Comment:
same here. collect all the policy ids and use the policyentitymgr method to validate policies